web/client/components/background/BackgroundSelector.jsx

```diff
@@ -39,6 +39,8 @@ class BackgroundSelector extends React.Component {
         onLayerChange: PropTypes.func,
         onStartChange: PropTypes.func,
         onAdd: PropTypes.func,
+        hasCatalog: PropTypes.bool,
```

Comment:
I have a question, With this we are binding the BackgroundSelector component to something else like a Catalog.
Haven't we always said that this should be something hidden from the Receiver perspective.
I know Stefano has did something similar where pluginSender inject tools in pluginReceiver.
My opinion is to discuss this in the next dev meeting

Reply:
sure, this has been done to avoid refactors of complex nested components inside these elements. I added a TODO for this. 
I'll also open an issue about these improvements for the future.